lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -657,6 +674,7 @@ def _build_student_data(cls, user_id, course_key, usage_key_str):
         student_data_keys = set()
 
         with store.bulk_operations(course_key):
+            base_path = cls._build_block_base_path(store.get_item(usage_key))
```

Comment:
Is building this path necessary to fix the bug? It's convenient, but it seems like a format change that might break client expectations?

Reply:
It is not necessary for this bug, but I thought it would be a small useful improvement.  It is a backwards-incompatible change, but I think it makes the paths more consistent. One main benefit I see here is when you need to combine two reports, the paths are more useful. 

I can definitely remove it, or move it to my other PR (https://github.com/edx/edx-platform/pull/19781) that deals with enhancements to this code.

Reply:
If the goal is to get the bugfix into Ironwood, then I think we should go for as much compatibility as possible, and put enhancements into a separate PR (either #19781 or something else).